### ISSUE TITLE
file_info_long() method is broken.

### ISSUE THREAD

**Issue by davidhsmith** [holgern/pyedflib#25]
it returns this error: 
```
C:\Users\davids\AppData\Local\Continuum\Anaconda3\lib\site-packages\pyedflib\edfreader.py in file_info_long(self)
    686         self.file_info()
    687         for ii in np.arange(self.signals_in_file):
--> 688             print("label:", self.getSignalLabel(ii), "fs:",
    689                   self.getSignalFreqs()[ii], "nsamples",
    690                   self.getNSamples()[ii])
```
AttributeError: 'EdfReader' object has no attribute 'getSignalLabel'

The fix seems to be to change the lines:
```python
    def file_info_long(self):
        self.file_info()
        for ii in np.arange(self.signals_in_file):
            print("label:", self.getSignalLabel(ii), "fs:",
                  self.getSignalFreqs()[ii], "nsamples",
                  self.getNSamples()[ii])
```
to:
```python
    def file_info_long(self):
        self.file_info()
        for ii in np.arange(self.signals_in_file):
            print("label:", self.getSignalLabels()[ii], "fs:",    # <==Note change here
                  self.getSignalFreqs()[ii], "nsamples",
                  self.getNSamples()[ii])
```
Sorry about the poor formatting of my issue.

**Comment by mattvend** [comment]
Same issue for me, I noticed the same problem.
On top of that, it seems that getSignalFreqs was renamed in getSampleFrequency.

**Comment by holgern** [comment]
Thanks for reporting, this issue is fixed in version 0.1.12
